### PR TITLE
Fixed mistakes in german locale file

### PR DESCRIPTION
**Pull request by w1tcherrr**
Fixed multiple mistakes/minor inaccuracies I made during the last translation.

### CODE REVIEW

File: src/main/resources/locale/locale_de.properties [hunk 1]
```diff
@@ -1088,9 +1088,9 @@ XPBar.Woodcutting = Holzf\u00E4llen Level: &6{0}
...
+mcMMO.Description = &3\u00DCber das &emcMMO&3 Projekt: &6mcMMO ist ein &copen source&6 RPG-Mod erstellt im Februar 2011 &6von &9nossr50&6. Das Ziel ist es ein qualitatives RPG Erlebnis zu liefern. &3Tips:&6 - &aNutze &c/mcmmo help&a um Befehle zu sehen, &6 - &aNutze &c/skillname&a f\u00FCr detaillierte Skill Infos, &3Entwickler:&6 - &anossr50 &9(Erfinder & Projektleitung),&6 - &aGJ &9(Fr\u00FChere Projektleitung),&6 - &aNuclearW &9(Entwickler),&6 - &abm01 &9(Entwickler),&6 - &aTfT_02 &9(Entwickler),&6 - &aGlitchfinder &9(Entwickler),&6 - &at00thpick1 &9(Entwickler),&6 - &alumis31 &9(Urspr\u00FCngliche Deutsche \u00DCbersetzung),&6 - &aOverCrave &9(Neue Deutsche \u00DCbersetzung & \u00DCberarbeitung),&6 - &aAnseba &9(\u00DCberarbeitung Deutsche \u00DCbersetzung), &3N\u00FCtzliche Links:&6 - &ahttps://github.com/mcMMO-Dev/mcMMO/issues&6 Bug Reporting,&6 - &ahttps://discord.gg/EJGVanb &6 Offizieller Discord (Englisch)
```
Comment (TheBusyBiscuit) [inline src/main/resources/locale/locale_de.properties:1091]: Missing 'p'.

Also I see you added yourself there :eyes:
```suggestion
mcMMO.Description = &3\u00DCber das &emcMMO&3 Projekt: &6mcMMO ist ein &copen source&6 RPG-Mod erstellt im Februar 2011 &6von &9nossr50&6. Das Ziel ist es ein qualitatives RPG Erlebnis zu liefern. &3Tipps:&6 - &aNutze &c/mcmmo help&a um Befehle zu sehen, &6 - &aNutze &c/skillname&a f\u00FCr detaillierte Skill Infos, &3Entwickler:&6 - &anossr50 &9(Erfinder & Projektleitung),&6 - &aGJ &9(Fr\u00FChere Projektleitung),&6 - &aNuclearW &9(Entwickler),&6 - &abm01 &9(Entwickler),&6 - &aTfT_02 &9(Entwickler),&6 - &aGlitchfinder &9(Entwickler),&6 - &at00thpick1 &9(Entwickler),&6 - &alumis31 &9(Urspr\u00FCngliche Deutsche \u00DCbersetzung),&6 - &aOverCrave &9(Neue Deutsche \u00DCbersetzung & \u00DCberarbeitung),&6 - &aAnseba &9(\u00DCberarbeitung Deutsche \u00DCbersetzung), &3N\u00FCtzliche Links:&6 - &ahttps://github.com/mcMMO-Dev/mcMMO/issues&6 Bug Reporting,&6 - &ahttps://discord.gg/EJGVanb &6 Offizieller Discord (Englisch)
```

File: src/main/resources/locale/locale_de.properties [hunk 2]
```diff
@@ -610,7 +610,7 @@ JSON.Type.Passive = Passiv
 JSON.Type.SuperAbility = Superf\u00E4higkeit
 JSON.URL.Discord = Der offizielle (englische) mcMMO Discord Server!
 JSON.URL.Patreon = Unterst\u00FCtze die Entwicklung von mcMMO \u00FCber nossr50's Patreon!
-JSON.URL.Spigot = Die offizielle mcmmo Spigot Seite
+JSON.URL.Spigot = Die offizielle mcmmo Spigot Seite.
```
Comment (TheBusyBiscuit) [inline src/main/resources/locale/locale_de.properties:613]:
```suggestion
JSON.URL.Spigot = Die offizielle mcMMO Spigot-Seite.
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by w1tcherrr** [comment]
Thanks you TheBusyBiscuit, I added both of your fixes and also the ones I missed from #4455.
I added the name "Anseba" into the file as that is my friend who translated/fixed most of the translations (#4423) (I mostly just uploaded and reviewed quickly since she doesn't have/use GitHub, my bad for not noticing the mistakes in the first place). I thought it would be nice to commemorate her work, but if that is not wanted I can simply delete that addition again ^^!

**Comment by TheBusyBiscuit** [comment]
> but if that is not wanted I can simply delete that addition again

I am not qualified to make a call on that, I am just translating the german messages here like you and tend to give my feedback on some things. I just noticed that change and thought it was a funny thing to point out XD

**Review by TheBusyBiscuit — APPROVED** [review]
The translations are looking good to me

**Comment by nossr50** [comment]
Thank you

**Pull request merged**